### PR TITLE
[Website] Minor doc fixes

### PR DESCRIPTION
**Pull request by akartsky**
- Change `releases` dropdown names
- Fix broken links/images
- Format text

### CODE REVIEW

File: website/content/en/docs/deployment/add-ons/cloudwatch/guide.md [hunk 1]
```diff
@@ -32,19 +32,19 @@ aws cloudwatch list-metrics --namespace ContainerInsights --region $CLUSTER_REGI
...
 
 For a full list of metrics that are provided by default, see [Amazon EKS and Kubernetes Container Insights metrics](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-metrics-EKS.html).
 
 The metrics are grouped by varying parameters such as Cluster, Namespace, or PodName.
-![cloudwatch-metrics](https://raw.githubusercontent.com/awslabs/kubeflow-manifests/main/deployments/add-ons/cloudwatch/images/cloudwatch-metrics.png)
+![cloudwatch-metrics](../../../../images/cloudwatch/cloudwatch-metrics.png)
 
 The following image is an example of the graphed metrics for the `istio-system` namespace that deals with internet traffic.
-![cloudwatch-namespace-metrics](https://raw.githubusercontent.com/awslabs/kubeflow-manifests/main/deployments/add-ons/cloudwatch/images/cloudwatch-namespace-metrics.png)
+![cloudwatch-namespace-metrics](../../../../images/cloudwatch/cloudwatch-namespace-metrics.png)
```
Comment (surajkota) [inline website/content/en/docs/deployment/add-ons/cloudwatch/guide.md:47]: doesnt it need to be in {{ref }}

File: website/content/en/docs/deployment/cognito/guide-automated.md [hunk 2]
```diff
@@ -72,12 +72,11 @@ This guide assumes you have Python 3.8 installed and that you have completed the
                     us-east-1-certARN: arn:aws:acm:us-east-1:123456789012:certificate/373cc726-f525-4bc7-b7bf-d1d7b641c238
             ```
 
-<<<<<<< HEAD
 1. Install Kubeflow using the following command:
    ```bash
     while ! kustomize build deployments/cognito | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 30; done
     ```
-
+
```
Comment (surajkota) [inline website/content/en/docs/deployment/cognito/guide-automated.md:79]: nit: do we need this whitespace?

File: website/content/en/docs/deployment/cognito/guide-automated.md [hunk 3]
```diff
@@ -119,7 +118,7 @@ This guide assumes you have Python 3.8 installed and that you have completed the
     kustomize build deployments/cognito | kubectl delete -f -
     ```
 1. To delete the rest of resources(subdomain, certificates etc.), run the following commands from the root of your repository:
-> Note: Make sure that you have the configuration file created by the script in `tests/e2e/utils/cognito_bootstrap/config.yaml`. If you did not use the script, plug in the name, ARN, or ID of the resources that you created in a yaml file in `tests/e2e/utils/cognito_bootstrap/config.yaml` by referring to the following sample:
+    > Note: Make sure that you have the configuration file created by the script in `tests/e2e/utils/cognito_bootstrap/config.yaml`. If you did not use the script, plug in the name, ARN, or ID of the resources that you created in a yaml file in `tests/e2e/utils/cognito_bootstrap/config.yaml` by referring to the following sample:
```
Comment (surajkota) [inline website/content/en/docs/deployment/cognito/guide-automated.md:121]: thx, can you also check the note in line 151?

File: website/content/en/docs/deployment/add-ons/cloudwatch/guide.md (hunk 1 above)
Comment (akartsky) [reply website/content/en/docs/deployment/add-ons/cloudwatch/guide.md:47]: ref is for .md files only

File: website/content/en/docs/deployment/cognito/guide-automated.md (hunk 3 above)
Comment (akartsky) [reply website/content/en/docs/deployment/cognito/guide-automated.md:121]: looks fine : https://akartsky.github.io/kubeflow-manifests/main/docs/deployment/cognito/guide-automated/

**Review by surajkota — APPROVED** [review]

**Pull request merged**